lib/moab/storage_object.rb

```diff
@@ -230,6 +226,18 @@ def restore_object(recovery_path)
       self
     end
 
+    # @return [Integer] the size occupied on disk by the storage object, in bytes.  this is the entire moab (all versions).
+    def size
+      size = 0
+      Find.find(object_pathname) do |path|
+        if FileTest.directory?(path)
+          Find.prune if File.basename(path)[0] == '.'
+        else
+          size += FileTest.size(path)
+        end
+      end
+      size
+    end
```

Comment:
Also prune `..` directory?

Reply:
AFAICT from a quick test on the rails console, neither of the implicit `.` or `..` dirs appears in the list of dirs that `Find.find` iterates over.